terraform/environments/cdpt-chaps/locals.tf

```diff
@@ -1,4 +1,5 @@
 locals {
+  app_data = jsondecode(file("./application_variables.json"))
```

Comment:
FYI, "platform_locals.tf" creates a variable called `application_data` with the same values by default